src/sass/parse.js

```diff
@@ -4836,15 +4846,19 @@ function checkCompoundSelector1(i) {
         checkAttributeSelector(i) ||
         checkPseudo(i) ||
         checkPlaceholder(i);
```

Comment:
No need as far as I'm aware. 

`checkClass`, `checkAttributeSelector`, `checkPseudo` and `checkPlaceholder` all catch interpolation if it follows one of them.

If we just have `h2#{$foo}` then it's picked up by the `checkTypeSelector` on line 4828.

The reason we have to check for interp in` checkCompoundSelector2` is that we can't rely on `checkTypeSelector` as it's invalid to have a typeSelector directly follow anything (without spaces) right?

Am I missing something?